client/state/reader/thumbnails/actions.js

```diff
@@ -45,33 +45,37 @@ export const requestThumbnail = ( embedUrl ) => ( dispatch ) => {
 		case 'videopress': {
 			const posterEndpoint = `https://public-api.wordpress.com/rest/v1.1/videos/${ id }/poster`;
 
-			return globalThis.fetch( posterEndpoint ).then( async ( response ) => {
-				let json;
-				try {
-					json = await response.json();
-				} catch ( error ) {}
+			try {
+				return globalThis.fetch( posterEndpoint ).then( async ( response ) => {
+					let json;
+					try {
```

Comment:
Do we still need this inner `try/catch` block?

Reply:
Good catch - nope, we don't. Cleaned them up in https://github.com/Automattic/wp-calypso/pull/79111/commits/44f2ea8f635c316c2f376ffc1ddd8bea6af7f18d